design/KafkaDesign.md

```diff
@@ -0,0 +1,420 @@
+	"status": "COMPLETED",    
+	"job_id": "65603c30-64ee-4bd2-85db-8328425c3b09"
+  },
+  "experiments": {
+	"prometheus-1|default|cadvisor|cadvisor(daemonset)|cadvisor": {
+  	"name": "prometheus-1|default|cadvisor|cadvisor(daemonset)|cadvisor",
+  	"status": "PROCESSED",
+  	"apis": {
+    	"recommendations": {
+      	"response": [
+        	{
+          	"cluster_name": "default",
+          	"experiment_type": "container",
+          	"kubernetes_objects": [
+            	{
+              	"type": "daemonset",
+              	"name": "cadvisor",
+              	"namespace": "cadvisor",
+              	"containers": [
+                	{
+                  	"container_image_name": "gcr.io/cadvisor/cadvisor:v0.45.0",
+                  	"container_name": "cadvisor",
+                  	"recommendations": {
+                    	"version": "1.0",
+                    	"notifications": {
+                      	"111000": {
+                        	"type": "info",
+                        	"message": "Recommendations Are Available",
+                        	"code": 111000
+                      	}
+                    	},
+                    	"data": {
+                      	"2025-01-27T11:28:24.000Z": {
+                        	"notifications": { 	},
+                        	"monitoring_end_time": "2025-01-27T11:28:24.000Z",
+                        	"current": {},
+                        	"recommendation_terms": {
+                          	"short_term": {
+                            	"duration_in_hours": 24,
+                            	"notifications": {
+                              	"112101": {
+                                	"type": "info",
+                                	"message": "Cost Recommendations Available",
+                                	"code": 112101
+                              	},
+                              	"112102": {
+                                	"type": "info",
+                                	"message": "Performance Recommendations Available",
+                                	"code": 112102
+                              	}
+                            	},
+                            	"monitoring_start_time": "2025-01-26T11:28:24.000Z",
+                            	"recommendation_engines": {
+                              	"cost": {
+                                	"pods_count": 1,
+                                	"confidence_level": 0,
+                                	"config": {
+                                  	"limits": {
+                                    	"memory": {
+                                      	"amount": 136651161.6,
+                                      	"format": "bytes"
+                                    	},
+                                    	"cpu": {
+                                      	"amount": 0.22053964085707395,
+                                      	"format": "cores"
+                                    	}
+                                  	},
```

Comment:
The entire JSON seems to be unformatted and difficult to read ... please make sure that it is formatted properly

Reply:
Done